spec/features/visitor_can_create_account_spec.rb

```diff
@@ -37,5 +37,89 @@
       expect(page).to have_content("Login")
       expect(page).to_not have_content("Logout")
     end
+    it "is not allowed to created account without username" do
+      visit root_path
+      click_link "Create Account"
+
+      fill_in "Password", with: 'password'
+      fill_in "First name", with: 'Ross'
+      fill_in "Last name", with: 'Edfort'
+      fill_in "Email address", with: 'fake@fake.com'
+      fill_in "Phone number", with: '111-111-1111'
+      click_button "Create Account"
+
+      expect(page).to have_content("Username cannot be blank")
+      expect(page).to_not have_content("Hello Ross!")
+    end
+    it "is not allowed to created account without password" do
+      visit root_path
+      click_link "Create Account"
+
+      fill_in "Username", with: "rossedfort"
+      fill_in "First name", with: 'Ross'
+      fill_in "Last name", with: 'Edfort'
+      fill_in "Email address", with: 'fake@fake.com'
+      fill_in "Phone number", with: '111-111-1111'
+      click_button "Create Account"
+
+      expect(page).to have_content("Password cannot be blank")
+      expect(page).to_not have_content("Hello Ross!")
+    end
+    it "is not allowed to created account without first name" do
+      visit root_path
+      click_link "Create Account"
+
+      fill_in "Username", with: "rossedfort"
+      fill_in "Password", with: 'password'
+      fill_in "Last name", with: 'Edfort'
+      fill_in "Email address", with: 'fake@fake.com'
+      fill_in "Phone number", with: '111-111-1111'
+      click_button "Create Account"
+
+      expect(page).to have_content("First name cannot be blank")
+      expect(page).to_not have_content("Hello Ross!")
+    end
+    it "is not allowed to created account without last name" do
+      visit root_path
+      click_link "Create Account"
+
+      fill_in "Username", with: "rossedfort"
+      fill_in "Password", with: 'password'
+      fill_in "First name", with: 'Ross'
+      fill_in "Email address", with: 'fake@fake.com'
+      fill_in "Phone number", with: '111-111-1111'
+      click_button "Create Account"
+
+      expect(page).to have_content("Last name cannot be blank")
+      expect(page).to_not have_content("Hello Ross!")
+    end
+    it "is not allowed to created account without email address" do
+      visit root_path
+      click_link "Create Account"
+
+      fill_in "Username", with: "rossedfort"
+      fill_in "Password", with: 'password'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.